pkgs/applications/networking/instant-messengers/pantalaimon/default.nix

```diff
@@ -5,6 +5,7 @@
 , installShellFiles
 , nixosTests
 , enableDbusUi ? true
+, wrapGAppsHook
```

Comment:
```suggestion
, wrapGAppsHook3
```

Reply:
oh that's  why ci was failing! thank you!